go.mod

```diff
@@ -1,9 +1,19 @@
 module github.com/in-toto/witness
 
-go 1.19
+go 1.21
+
+toolchain go1.21.4
+
+replace github.com/in-toto/go-witness => ../go-witness
```

Comment:
you should checkout go workspaces! https://go.dev/doc/tutorial/workspaces

Reply:
TIL, this is very handy!

Reply:
100% - need to put it on my TODOs